src/pages/AboutPage.js

```diff
@@ -1,5 +1,140 @@
+import styled from "styled-components";
```

Comment:
All css / styled components må inn i egen fil

Reply:
Husk å lage en egen mappe for AboutUsPage hvor du har alle filene og tingene som brukes der